src/frontend/Komponenter/Behandling/SettPåVent/LokalkontorOppgavevalg.tsx

```diff
@@ -29,14 +30,23 @@ type Props = {
     settOppgaverMotLokalkontor: Dispatch<SetStateAction<VurderHenvendelseOppgavetype[]>>;
     oppgaverMotLokalkontor: VurderHenvendelseOppgavetype[];
     erBehandlingPåVent: boolean;
+    settInnstillingsoppgaveBeskrivelse: Dispatch<SetStateAction<string>>;
+    erKnappSattPåVentTrykket: boolean;
 };
 
+const ReadMoreWrapper = styled(ReadMore)`
+    max-width: 45rem;
+    margin-top
+`;
+
 export const LokalkontorOppgavevalg: FC<Props> = ({
     aktuelleOppgaver,
     sendteOppgaver,
     settOppgaverMotLokalkontor,
     oppgaverMotLokalkontor,
     erBehandlingPåVent,
+    settInnstillingsoppgaveBeskrivelse,
+    erKnappSattPåVentTrykket,
```

Comment:
Litt vanskelig å skjønne hva som menes her, og hva det hjelper oss?

Blir denne "true" når man har satt en behandling på vent? Slik det er nå så vil den staten forsvinne når man refresher siden... Da er det bedre å bruke tilstanden til behandlingen for å sjekke dette?

Men hvis behovet er å bytte fra lese/skrivemodus så kanskje vi må sjekke om det finnes et lokalkontorvalg fra før, eller?

Reply:
Staten var kun med for å skjule "legg til beskjed" alternativet som inneholder en tekstboks. Det er nå oppdatert slik at dette gjøres på en annen måte.